examples/slurm/README.md

```diff
@@ -0,0 +1,215 @@
+# SLURM-based HPC Cluster Setup Guide
+This guide will help you deploy ServerlessLLM on a SLURM-based HPC (High Performance Computing) cluster. Please make sure you have installed the ServerlessLLM following the [installation guide](https://serverlessllm.github.io/docs/stable/getting_started/installation#installing-with-pip) on all machines.
+## Some Tips about Installation
+Firstly, **all installation work should be done on the login node**. We recommend using [installing with pip](https://serverlessllm.github.io/docs/stable/getting_started/installation#installing-with-pip) if there is no CUDA driver on login node.
```

Comment:
Previously I did create an installation script and submit it to the node to run. But I got an error, like
```shell
[1/9] Creating directories for 'gflags-populate"
[1/9] Performing download step (git clone) for "gflags-populate"
Cloning into 'gflags-src'.
fatal: unable to access 'https://github.com/gflags/gflags.git/': Failed to connect to github.com po rt 443: Connection timed out
Cloning into 'gflags-src'.
fatal: unable to access 'https://github.com/gflags/gflags.git/': Failed to connect to github.com po rt 443: Connection timed out
...
```
This error located in ```sllm_store/CMakeLists.txt```, it needs network access to clone some repositories from github. However, job node's 443 port is blocked, leading to an error.

This problem only occurs when ```sllm_store``` is installed, not when ```sllm``` or ```sllm[worker]``` is installed. So submitting the install script to a job node to install ```sllm``` and ```sllm[worker]``` is possible, but ```sllm_store``` can only be installed on the login node at the moment. 

By the way, do I need to clarify these cases in the documentation as well?

Reply:
It's only because of the settings of a specific node. Not all slurm nodes will block 443 port. It would be better to mention that both install and build needs network connection, compared with "must on head node".